pages/games/[hash].js

```diff
@@ -1,34 +1,80 @@
-import styles from "../../styles/Game.module.css";
+import { useState } from "react";
 
 function Game({ data }) {
+  const PLAYER = "PLAYER";
```

Comment:
You can place these constants outside the component to avoid generating them each time the component is rendered.

Reply:
Did not think about that. Nice tip :+1: